src/components/GitHubProfilePictureExtended.svelte

```diff
@@ -13,13 +13,13 @@
 </script>
 
 <GitHubProfilePicture
-    wrapperClasses="btn btn-light rounded-pill mb-2 p-0 pe-2 d-flex align-items-center"
+    wrapperClasses="btn btn-light rounded-pill mb-2 p-0 pe-2 d-inline-flex align-items-center"
     image={'https://github.com/' + username + '.png'}
     name={username}
     circle={true}
     size={50}
 >
-    <div class={'ms-2 d-inline-block text-start d-flex flex-column align-items-center px-2 ' + extra_classes}>
+    <div class={'ms-2 d-inline-block text-start d-flex flex-column align-items-left px-2 ' + extra_classes}>
```

Comment:
```suggestion
    <div class={'ms-2 d-inline-block text-start d-flex flex-column align-items-start px-2 ' + extra_classes}>
```
it's bootstrap 5 😉

Reply:
Good job it's the default 😅 

Honestly one of my biggest annoyances with bootstrap.